.github/workflows/updater.yml

```diff
@@ -0,0 +1,15 @@
+name: Update S3
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Should probably be develop? `master` is only for production, right?

Reply:
Yes, this should only be run when deploying to production.